contracts/Peeranha.sol

```diff
@@ -99,7 +103,7 @@ contract Peeranha is IPeeranha, Initializable {
      * - Must be an community.  
      */
     function followCommunity(address userAddress, uint32 communityId) external override 
-    onlyExistingAndNotFrozenCommunity(communityId) {
+    onlyExistingAndNotFrozenCommunity(communityId) {  // isDelegateUserAndExisitingUser(userAddress); check deligate?
```

Comment:
This is another place where we need to create a used if does not exist. And for now allow delefate user to execute.